src/app/[locale]/client-filters.tsx

```diff
@@ -112,9 +122,85 @@ export default function ClientFilters({
             testId="publishersFilter"
           />
         )}
-      </PageSidebar>
+      </>
+    ),
+    [
+      filters.licenses,
+      filters.publishers,
+      query,
+      selectedLicenses,
+      selectedPublishers,
+      t,
+    ]
+  );
+
+  return (
+    <>
+      {/* Small screen filter dialog */}
+      <Transition.Root show={showFiltersSidebarOnSmallScreen} as={Fragment}>
+        <Dialog
+          as="div"
+          className="relative z-40 lg:hidden"
+          onClose={setShowFiltersSidebarOnSmallScreen}
+        >
+          <Transition.Child
+            as={Fragment}
+            enter="transition-opacity ease-linear duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="transition-opacity ease-linear duration-300"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <div className="fixed inset-0 bg-black bg-opacity-25" />
+          </Transition.Child>
+
```

Comment:
Minor: when do we use a newline to separate tags? This line uses one, but e.g. line 172-173 doesn't.